setup.cfg

```diff
@@ -5,5 +5,4 @@ license_file = LICENSE.txt
 [flake8]
 # define the flake8 configurations
 exclude = .nox/
-max-complexity = 10

```

Comment:
I had to remove this. I think we don't need it. However, I can restore it in the future.